### PR TITLE
Fix to parsePacket, getServiceName and addition of test suite.

### PR DESCRIPTION
**Pull request by Recodify**

### CODE REVIEW

**Comment by grahamlyons** [comment]
@joshgummersall are you happy to merge this pull request?

File: test/riemannBackend.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+var test = require("unit.js");
+var config = { riemann: { host: {} } };
+var riemann = new require('../lib/riemann');
```
Comment (joshgummersall) [inline test/riemannBackend.js:3]: should not need the `new` here

File: test/riemannBackend.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		});
+
+		it("should return the namespace without the metric and type, when parseNamespace is true", function(){
+			config.riemann.parseNamespace = true;
+			var serviceName = riemannBackend.getService("ns.foo:1|c\n");
+			config.riemann.parseNamespace = false;
+			test.string(serviceName).is("ns");
+		});
+	});
+})
```
Comment (joshgummersall) [inline test/riemannBackend.js:34]: add semicolon

File: test/riemannBackend.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+var test = require("unit.js");
+var config = { riemann: { host: {} } };
+var riemann = new require('../lib/riemann');
+
+describe("riemann-backend", function(){	    
+
+	var riemannBackend = new riemann({}, config, { on: function(){ } });
```
Comment (joshgummersall) [inline test/riemannBackend.js:7]: If you don't mind just fixing the tabs here (spaces instead of tabs and two spaces per tab that would be great.

**Comment by joshgummersall** [comment]
@samshiles and @grahamlyons couple of quick style comments. looks good though! thanks for contributing, much appreciated!

**Comment by psanford** [comment]
What's the state of this pull request? Can we get this merged?

**Comment by joshgummersall** [comment]
I left a few comments for @samshiles to fix before merging.

**Pull request merged**